### PR TITLE
Clarify zero-value behavior of `ctlz`/`cttz` intrinsics.

### PR DESCRIPTION
**Pull request by frewsxcv**
Fixes https://github.com/rust-lang/rust/issues/34381.

### CODE REVIEW

**Comment by frewsxcv** [comment]
Hold off on reviewing. Going to discuss this at this week's docs team meeting.

**Comment by aturon** [comment]
@frewsxcv Ok, let me know when you're ready.

File: src/libcore/intrinsics.rs [hunk 1]
```diff
@@ -1156,10 +1156,44 @@ extern "rust-intrinsic" {
     /// Returns the number of bits set in an integer type `T`
     pub fn ctpop<T>(x: T) -> T;
 
-    /// Returns the number of leading bits unset in an integer type `T`
+    /// Returns the number of leading unset bits (zeroes) in an integer type `T`. Unlike the
+    /// underlying LLVM intrinsic, an `x` with value `0` will always return the bit width of `T`.
```
Comment (frewsxcv) [inline src/libcore/intrinsics.rs]: Removed that sentence in the latest force push.

**Comment by frewsxcv** [comment]
@aturon @nagisa This is ready for review now.

**Pull request merged**